conanfile.py

```diff
@@ -87,24 +88,63 @@ def configure(self):
+        self.options["ffmpeg"].with_libaom = False
+        self.options["ffmpeg"].with_libsvtav1 = False
+        self.options["ffmpeg"].with_libdav1d = False
+        self.options["ffmpeg"].with_zlib = False
+        self.options["ffmpeg"].with_bzip2 = False
+        self.options["ffmpeg"].with_lzma = True
+        self.options["ffmpeg"].with_libiconv = False
         self.options["ffmpeg"].with_freetype = False
-        self.options["ffmpeg"].with_libfdk_aac = False
-        self.options["ffmpeg"].with_libmp3lame = False
-        self.options["ffmpeg"].with_libvpx = False
-        self.options["ffmpeg"].with_libwebp = False
-        self.options["ffmpeg"].with_libx264 = False
-        self.options["ffmpeg"].with_libx265 = False
-        self.options["ffmpeg"].with_openh264 = False
         self.options["ffmpeg"].with_openjpeg = False
+        self.options["ffmpeg"].with_openh264 = False
         self.options["ffmpeg"].with_opus = False
-        self.options["ffmpeg"].with_programs = False
-        self.options["ffmpeg"].with_ssl = False
         self.options["ffmpeg"].with_vorbis = False
+        self.options["ffmpeg"].with_libx264 = False
+        self.options["ffmpeg"].with_libx265 = False
+        self.options["ffmpeg"].with_libvpx = False
+        self.options["ffmpeg"].with_libmp3lame = False
+        self.options["ffmpeg"].with_libwebp = False
+        self.options["ffmpeg"].with_ssl = False
+        if self.settings.os != "Android":
+            self.options["ffmpeg"].with_libfdk_aac = False
+
+        self.options["ffmpeg"].avcodec = True
+        self.options["ffmpeg"].avformat = True
+        self.options["ffmpeg"].swscale = True
+        # We want following options supported:
+        # H3:SoD - .bik and .smk
+        # H3:HD  -  ogg container / theora video / vorbis sound (not supported by vcmi at the moment, but might be supported in future)
+        # and for mods - webm container / vp8 or vp9 video / opus sound
+        # TODO: add av1 support for mods (requires enabling libdav1d which currently fails to build via Conan)
+        self.options["ffmpeg"].enable_protocols = "file"
+        self.options["ffmpeg"].enable_demuxers = "bink,binka,ogg,smacker,webm_dash_manifest"
+        self.options["ffmpeg"].enable_parsers = "opus,vorbis,vp8,vp9,webp"
+        self.options["ffmpeg"].enable_decoders = "bink,binkaudio_dct,binkaudio_rdft,smackaud,smacker,theora,vorbis,vp8,vp9,opus"
+
+        self.options["ffmpeg"].with_programs = False
+        self.options["ffmpeg"].avfilter = False
+        self.options["ffmpeg"].with_sdl = False
+
+        #optionally, for testing - enable ffplay/ffprobe binaries in conan package:
+        #if self.settings.os == "Windows":
+        #    self.options["ffmpeg"].with_programs = True
+        #    self.options["ffmpeg"].avfilter = True
+        #    self.options["ffmpeg"].with_sdl = True
+        #    self.options["ffmpeg"].enable_filters = "aresample,scale"
```

Comment:
FYI you can pass it on command line

Reply:
I used it on CI, so no command line.

Another problem was to find correct set of switches. Simple 'with_programs' would silently fail, had to dig into ffmpeg's configure to figure out that it also needs avfilter & with_sdl. So I'd rather keep it documented somewhere.